include/prime_generator.h

```diff
@@ -1,28 +1,40 @@
 #pragma once

```

Comment:
I've noticed you switched from the AKS primality test to the naive square root test. I would guess that this is to accommodate the extended integer types. Is that correct?

Reply:
Hmm... I'm looking over our code and I don't think we ever used AKS. From what I can tell, the IsPrime function has always been the naive square root test and all the other things in aks.h/cpp (LargestPrimeFactor, Powm, Inv, CPolyMod) have never been used. The files aks.h and aks.cpp were created in commit 7ef0632aa6536df1b01afe50ae492330f21011a9, and they were using the naive primality test back then.

Reply:
We haven't been using AKS to do primality testing. The `IsPrime` function we'd been using is essentially a brute-force prime checker.